src/System.Management.Automation/engine/regex.cs

```diff
@@ -86,15 +86,17 @@ internal string PatternConvertedToRegex
         /// </summary>
         /// <param name="pattern">The wildcard pattern to match.</param>
         /// <returns>The constructed WildcardPattern object.</returns>
-        /// <remarks> if wildCardType == None, the pattern does not have wild cards</remarks>
+        /// <remarks>If wildCardType == None, the pattern does not have wild cards.</remarks>
         public WildcardPattern(string pattern)
```

Comment:
Since you are here, how about making this constructor calling the other one?
```
public WildcardPattern(string pattern) : this(pattern, WildcardOptions.None)
{
}
```
Then also, remove `= WildcardOptions.None` from `internal WildcardOptions Options { get; } = WildcardOptions.None;`.

Reply:
Fixed.